Postprocessing/metrics/Deadwood.json

```diff
@@ -0,0 +1 @@
+{"Deadwood, Tropical Dry": {"pool_tc_sum_MEAN": "15624317.95", "area_sum_MEAN": "1142790.38", "pool_tc_per_ha_MEAN": "13.67"}, "Deadwood, Tropical Moist": {"pool_tc_sum_MEAN": "15802289.91", "area_sum_MEAN": "608498.41", "pool_tc_per_ha_MEAN": "25.97"}, "Deadwood, Tropical Premontane Wet": {"pool_tc_sum_MEAN": "13011268.76", "area_sum_MEAN": "417245.77", "pool_tc_per_ha_MEAN": "31.18"}}
```

Comment:
This is cool. Does it refer to the mean over the lifetime of the simulation? Alternative/additional summary statistics might be start, mid- and endpoints. (e.g 1900, 1950, 2000) or decadal timestamps.

Reply:
I wrote a simple script that calculates the mean value of for each indicator for each LifeZone so we can use it for dvc metrics. But I didn't use it yet cause there were some inconcistencies.

Reply:
> This is cool. Does it refer to the mean over the lifetime of the simulation? Alternative/additional summary statistics might be start, mid- and endpoints. (e.g 1900, 1950, 2000) or decadal timestamps.

So,  should I use the `Pools_DOM_Sensitivity_forest_<[0,50,100]>_years.csv` files to calculate mean values ?